en/docs/develop/customizations/creating-custom-mediators.md

```diff
@@ -1,8 +1,8 @@
 # Creating a Custom Mediator
 
-If you need to create a custom mediator that performs some logic on a message, you can either create a new mediator project, or import an existing mediator project using WSO2 Integration Studio.
+If you need to create a custom mediator that performs some logic on a message, you can either create a new project for mediator, or import an existing mediator project using MI for VS Code.
```

Comment:
There is no separate project for custom mediators. You have to select 'Class Mediator'